### PR TITLE
[Feat] ChainLink Price Feed

### PR DESCRIPTION
**Pull request by MartnLecam**
# Name of the feature/issue
Add ChainLink price feed for ETH/USD

## Description
Add ChainLink to provide price for ETH/USD. No matter which network, it should always use ChainLink price.

## Type of change

Click the correct/s option/s

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  This change requires a documentation update

### CODE REVIEW

File: packages/background/src/controllers/ExchangeRatesController.ts [hunk 1]
```diff
@@ -18,6 +18,507 @@ import { ActionIntervalController } from './block-updates/ActionIntervalControll
...
+
+        return response.data[currencyApiId][currency];
+    },
+};
+
+const rateProvider: Record<string, RateService> = {
+    'usd-eth': chainLinkService,
+};
+
+const getRateService = (
```
Comment (fpezzola) [inline packages/background/src/controllers/ExchangeRatesController.ts:514]: I would put all the service related code in a separated file

File: packages/background/src/controllers/ExchangeRatesController.ts [hunk 2]
```diff
@@ -135,19 +647,18 @@ export class ExchangeRatesController extends BaseController<ExchangeRatesControl
...
-        const nativeCurrencyRates = (
-            await this._getNetworkNativeCurrencyRate()
-        )[currencyApiId];
+        const nativeCurrency =
+            this._preferencesController.nativeCurrency.toLowerCase();
 
-        // Response -> { ethereum : { [nativeCurrency] : rate }}
-        rates[symbol] =
-            nativeCurrencyRates[this._preferencesController.nativeCurrency];
+        rates[symbol] = await this._exchangeRateService.getRate(
```
Comment (fpezzola) [inline packages/background/src/controllers/ExchangeRatesController.ts:655]: What happens if the `chainLinkService` returns 0? Shouldn't we try the `coingeko` implementation.

**Review by fpezzola — APPROVED** [review]
LGTM!

**Review by julianariel — APPROVED** [review]
LGTM, good work 🥇

**Pull request merged**